src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/index.tsx

```diff
@@ -444,7 +444,7 @@ const StyledSearchBar = styled(SearchBar)`
     border-bottom-left-radius: 0;
   }
   .search-clear-form,
-  .icon-search {
+  [data-test-id='icon-search'] {
```

Comment:
Not too happy with this but it felt less bad than retaining a classname. I'd like to revisit the SearchBar component and fix the styling and element layout to work like SmartSearchBar. Right now they have different icons SearchBar's 🔍 icon is not clickable.

Reply:
I think a class name would be preferable in this case as this is a test attribute. (Maybe use something not prefixed with `icon`)